client/landing/gutenboarding/onboarding-block/stepper-wizard.tsx

```diff
@@ -0,0 +1,33 @@
+/**
+ * External dependencies
+ */
+import React, { useState } from 'react';
+
+export interface StepInputProps {
+	isActive: boolean;
+	onExpand: () => void;
+	onSelect: () => void;
+	inputClass: string;
+}
+
+interface Props {
+	children: React.ReactChildren;
+}
+
+export default function StepperWizard( props: Props ) {
+	const [ activeStep, setActiveStep ] = useState( 0 );
+
+	const handleNext = () => setActiveStep( activeStep + 1 );
+
+	return React.Children.map(
+		props.children,
+		( child, index ) =>
+			child &&
+			React.cloneElement( child, {
+				isActive: index === activeStep,
+				onExpand: () => setActiveStep( index ),
+				onSelect: handleNext,
+				inputClass: 'onboarding-block__question-input',
```

Comment:
Could this be the reason for [this style change](https://github.com/Automattic/wp-calypso/pull/37491#issuecomment-553886928)?

Reply:
No, this was just a way to have the input class as a constant. I was tempted to introduce an `<QuestionInput />` component but I dropped the idea in the end since it wasn't doing more than adding this class.